docs/source/gateway/index.rst

```diff
@@ -1,6 +1,1049 @@
+            openai_api_key: $OPENAI_API_KEY
+
+      - name: chat
+        type: llm/v1/chat
+        model:
+          provider: openai
+          name: gpt-3.5-turbo
+          config:
+            openai_api_base: https://api.openai.com/v1
+            openai_api_key: $OPENAI_API_KEY
+
+      - name: embeddings
+        type: llm/v1/embeddings
+        model:
+          provider: openai
+          name: text-embedding-ada-002
+          config:
+            openai_api_base: https://api.openai.com/v1
+            openai_api_key: $OPENAI_API_KEY
+
+Save this file to a location on the system that is going to be running the MLflow AI Gateway server.
+
+Step 4: Start the Gateway Service
+---------------------------------
+You're now ready to start the Gateway service!
+
+Use the ``MLflow AI Gateway start`` command and specify the path to your configuration file:
+
+.. code-block:: sh
+
+    MLflow AI Gateway start --config-path config.yaml --port {port} --host {host} --workers {worker count}
+
+If you do not specify the host, a localhost address will be used.
+
+If you do not specify the port, port 5000 will be used.
+
+The worker count for gunicorn defaults to 2 workers.
+
+Step 5: Access the Interactive API Documentation
+------------------------------------------------
+The MLflow AI Gateway service provides an interactive API documentation endpoint that you can use to explore
+and test the exposed routes. Navigate to ``http://{host}:{port}/`` (or ``http://{host}:{port}/docs``) in your browser to access it.
+
+The docs endpoint allow for direct interaction with the routes and permits submitting actual requests to the
+provider services by click on the "try it now" option within the endpoint definition entry.
+
+Step 6: Send Requests to Routes via REST API
+--------------------------------------------
+You can now send requests to the exposed routes. Here's an example of how to send a request using Python:
+
+**Note:** Replace "http://localhost:5000/gateway/routes/completions" with the URL of your Gateway service and
+the route you want to send a request to. You can replace completions with chat or embeddings to send requests to those routes.
+
+.. code-block:: python
+
+    import requests
+
+    data = {
+        "prompt": "Is this a test?",
+    }
+
+    response = requests.post(
+        "http://localhost:5000/gateway/routes/completions",
+        json=data,
+    )
+
+    print(response.json())
+
+The response printed to ``stdout`` will be:
+
+.. code-block:: python
+
+    {
+        "candidates": [
+            {
+                "message": {"role": "assistant", "content": "\n\nYes, this is a test."},
+                "metadata": {"finish_reason": "stop"},
+            }
+        ],
+        "metadata": {
+            "input_tokens": 13,
+            "output_tokens": 7,
+            "total_tokens": 20,
+            "model": "gpt-3.5-turbo-0301",
+            "route_type": "llm/v1/chat",
+        },
+    }
```

Comment:
I think we can remove this and instead tell users that they can also make REST requests & link them to the REST section